run-test.sh

```diff
@@ -128,6 +128,12 @@ create_test_overlay()
 	exit 1
   fi
   find $CoreFxBins -name '*.dll' -exec cp '{}' "$OverlayDir" ";"
+
+  # Then the native linux CoreFX binaries
+  #
+  # TODO: Currently, CI does not build the native CoreFX components so build them here
+  #       in the test phase for now.
+  ( $ProjectRoot/src/Native/build.sh && cp $ProjectRoot/bin/$OS.x64.$Configuration/Native/*.so $OverlayDir ) || exit 1
```

Comment:
This is fine for now.  As soon as this gets checked in I'll add the necessary legs.  Do you need new parameters in run-test to indicating the location of the corefx Linux native binaries, or will that fall under the corefx Linux managed binaries?